src/clickfunnels/main.ts

```diff
@@ -82,6 +103,25 @@ document.addEventListener('DOMContentLoaded', () => {
     return originalFetch.apply(this, args);
   };
 
+  // Attach listeners to all buttons currently on the page
+  document.querySelectorAll('.elButton').forEach((element: HTMLElement) => {
+    addButtonClickListener(element);
+  });
```

Comment:
Not important at all, just an FYI: since the expected argument to `forEach` has the same method signature as `addButtonClickListener`, you can omit the anonymous function that you're wrapping around it.

```suggestion
  document.querySelectorAll('.elButton').forEach(addButtonClickListener);
```